<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>500</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>680</width>
      <height>260</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>100</red>
       <green>100</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Game time:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumberMin">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clockDots">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberSec">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="atobButton">
       <property name="text">
        <string>A&lt;-&gt;B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="atocButton">
       <property name="text">
        <string>A&lt;-&gt;C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btocButton">
       <property name="text">
        <string>B&lt;-&gt;C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="infoBrowser">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>2.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>3.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>4.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>5.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="move1Label">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="move2Label">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="move3Label">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="move4Label">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="move5Label">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="newgameButton">
       <property name="text">
        <string>New game</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Start/Pause</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="discLabel">
         <property name="text">
          <string>Disc amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="disc_amountSpinbox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last 5 moves:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>181</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Top time</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>with</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="discamount2Label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>discs:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="toptimeLabel">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total moves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum moves</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalmovesLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="minmovesLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>with</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="discamountLabel">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>discs:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
